tests/pulseaudio_dbus_spec.lua

```diff
@@ -202,4 +203,19 @@ b.describe("PulseAudio with DBus", function ()
 			       end
 			   end
 		   end)
+
+		   b.it("Will Cycle through all available PlaybackStreams and move them to the FallbackSink", function()
+		   	   if #core.PlaybackStreams == 0 then
+				   print("\nWARNING: Can't cycle through all available PlaybackStreams and move them to the FallbackSink because there are no PlaybackStreams in this machine")
+				   return
+			   else
+				   stream = {}
+			   	   for ps=1,#core.PlaybackStreams do
+					   stream[ps] = {}
+					   stream[ps].path = core.PlaybackStreams[ps]
```

Comment:
As already mentioned for sinks, you don't need to do this, the object path is available as the `object_path` property on the playback stream itself.